develop-docs/self-hosted/email.mdx

```diff
@@ -11,14 +11,18 @@ description: Set up and configure email notifications for your self-hosted Sentr
 
 ## Outbound Email
 
-Self-hosted Sentry ships with a built-in outgoing SMTP server powered by [exim4](https://hub.docker.com/r/tianon/exim4). The default configuration is set to use this server. All you need to do is to set a valid address for `mail.from` setting in `config.yml` and the [FQDN](https://en.wikipedia.org/wiki/Fully_qualified_domain_name) of your Sentry instance for `SENTRY_MAIL_HOST` in `.env`. Keep in mind that if you start sending too many emails to public addresses, your new server may get marked as a spammer and banned.
+It is recommended to use an external SMTP server due to various sender requirements implemented by major email providers, but if you are testing or running a self-hosted Sentry instance on a local network, you can use the built-in SMTP serverpowered by [exim4](https://hub.docker.com/r/tianon/exim4). The default configuration is set to use this server. All you need to do is to set a valid address for `mail.from` setting in `config.yml` and the [FQDN](https://en.wikipedia.org/wiki/Fully_qualified_domain_name) of your Sentry instance for `SENTRY_MAIL_HOST` in `.env`.
```

Comment:
```suggestion
It is recommended to use an external SMTP server due to various sender requirements implemented by major email providers, but if you are testing or running a self-hosted Sentry instance on a local network, you can use the built-in SMTP server, powered by [exim4](https://hub.docker.com/r/tianon/exim4). The default configuration is set to use this server. All you need to do is to set a valid address for `mail.from` setting in `config.yml` and the [FQDN](https://en.wikipedia.org/wiki/Fully_qualified_domain_name) of your Sentry instance for `SENTRY_MAIL_HOST` in `.env`.
```

Reply:
Do we really need both `SENTRY_MAIL_HOST` and the `mail.from` config setting? If yes, is there no way to unify them? Or if these are separate (one for the SMTP server, the other for the from email part, we should clarify that in this text)

Reply:
> Do we really need both `SENTRY_MAIL_HOST` and the `mail.from` config setting? If yes, is there no way to unify them? Or if these are separate (one for the SMTP server, the other for the from email part, we should clarify that in this text)

I never use SENTRY_MAIL_HOST to be honest. It's always left blank.

Reply:
>I never use SENTRY_MAIL_HOST to be honest. It's always left blank.

Shall we clarify that?